src/controllers/routes.ts

```diff
@@ -1,12 +1,14 @@
 import { Router } from '@well-known-components/http-server'
 import { GlobalContext } from '../types'
-import { squidsHandler } from './handlers/squid-handler'
+import { listSquidsHandler, promoteSquidHandler, stopSquidHandler } from './handlers/squid-handler'
 
 // We return the entire router because it will be easier to test than a whole server
 export async function setupRouter(): Promise<Router<GlobalContext>> {
   const router = new Router<GlobalContext>()
 
-  router.get('/list', squidsHandler)
+  router.get('/list', listSquidsHandler)
+  router.get('/:id/promote', promoteSquidHandler)
```

Comment:
What about using `.post` to `promote` and `stop` actions?

Reply:
updated to PUT